posts/2020/12/jupyterlab-winter-theme.md

```diff
@@ -0,0 +1,233 @@
+tend to remind me of the holiday season, in the diff afer see how I change the
+brands colors from blue to red:
+
+```diff
+--- a/style/variables.css
++++ b/style/variables.css
+@@ -168 +168 @@                                                                  
+-  --jp-content-link-color: var(--md-blue-700);
++  --jp-content-link-color: var(--md-red-700);
+@@ -220,4 +220,4 @@                                                                  
+-  --jp-brand-color0: var(--md-blue-700);
+-  --jp-brand-color1: var(--md-blue-500);
+-  --jp-brand-color2: var(--md-blue-300);
+-  --jp-brand-color3: var(--md-blue-100);
++  --jp-brand-color0: var(--md-red-700);
++  --jp-brand-color1: var(--md-red-500);
++  --jp-brand-color2: var(--md-red-300);
++  --jp-brand-color3: var(--md-red-100);
+```
+
+This will affect most of the links ad many icons already. The input prompt in a
+notebook are still blue. Using my browser inspector I can find that this is
+controlled by the following which I change to red using one of the above
+variable.
+
+```
+@@ -274 +274 @@                                                                 .
+-  --jp-cell-inprompt-font-color: #307fc1;
++  --jp-cell-inprompt-font-color: var(--jp-brand-color1);
+```
+
+I'd like the "running notebook" dot in the filesystem browser to now be a
+snowman instead of a blue dot. Again using the inspector I can look a the css
+doing this and override it in my theme:
+
+
+```
+@@ -368,0 +369,26 @@                                                                 .
++
++.jp-DirListing-item.jp-mod-running .jp-DirListing-itemIcon:before {
++    content: '\2603'; /* snowman */
++    font-size: 10px;
++    position: absolute;
++    left: -8px;
++}
+```
+
+I also want the "notebook" and "json" file icons to be green, and the "shutdown
+kernel" button to be red:
+
+```
++g.jp-icon-warn0.jp-icon-selectable, g.jp-icon-warn1.jp-icon-selectable {
++    fill: var(--md-green-700);
++
++}
++
++button.jp-RunningSessions-itemShutdown.jp-mod-styled {
++    fill: var(--md-red-700);
++
++}
+```
+
+Let me also put images in the bottom left corner of many panels, I use a
+transparent white layer (FFFC gradient) on top of an image I do not forget to
+put in the `style/` directory, I can reference it from my css file:
+
+RIGHT NOW QS LOGO, BUT MAYBE USE A TREE, REINDEER...
+
+```
++.jp-Notebook, .jp-Launcher-body, .jp-DirListing-content {
++    background-image: linear-gradient(#FFFC,#FFFC),url(./logo.jpg);
++    background-repeat: no-repeat;
++    background-size: 120px 120px;
++    background-position: bottom left;
++}
+```
+
+
+
+
+# Awesome ! 
+
+Here is the final result, JupyterLab-winter-2020 provided by QuanSight, feel
+free to modify it, and please suggest some themes you might like and share your
+in the comments. For example we'd love to see a "summer 2020 theme" for our southern hemisphere friends. 
+We will dive into how to distribute your themes and make it high quality in a later blog post.
+
+
```

Comment:
I would like to link to whatever tag we decide for this series so it's easier for people to find them. I've proposed `JupyterTutorials` as a solution for now.

```suggestion
---
*This is part of a series of Jupyter tutorials. Find more [tutorials here](https://labs.quansight.org/categories/JupyterTutorials).*
```